example/ck_tile/01_fmha/fmha_fwd.cpp

```diff
@@ -1082,6 +1099,18 @@ bool run(const ck_tile::ArgParser& arg_parser)
 
             return fmha_fwd_splitkv(fmha_splitkv_traits, fmha_splitkv_args, stream_config);
         }
+#endif
+#if CK_TILE_FMHA_FWD_PAGEDKV_API
+        if(use_kvcache)
```

Comment:
If we enable both CK_TILE_FMHA_FWD_SPLITKV_API & CK_TILE_FMHA_FWD_PAGEDKV_API. We might invoke both of `fmha_fwd_splitkv()` and `fmha_fwd_pagedkv()` if `1 < num_splits && use_kvcache` is **true**

Reply:
In this case, I think it can make the splitkv kernel run because if num_stiles>1, the user clearly wants to run splitkv